bedrock/mozorg/templates/mozorg/home/home.html

```diff
@@ -253,7 +253,17 @@ <h2 class="primary go">{{ _('Discover <br>what drives our mission') }}</h2>
           </a>
         </li>
       {% endif %}
-      {% if l10n_has_tag('mdn_10_years') %}
+      {% if LANG == 'en-US' %}
+        <li id="promo-11" class="item promo-large-landscape viewsource" tabindex="0">
+          <h2 class="primary go">{{ _('Build the Web of tomorrow, today') }}</h2>
+          <a class="panel-link" href="https://ti.to/mdn/view-source-2015" data-promotion-id="promo-11" data-promotion-name="Mozilla Developer Network" data-promotion-class="promo-large-portrait" data-promotion-type="tile">
```

Comment:
Yeah, I actually did not receive an answer on that and kinda assumed that, seeing the text states "Register Now", that might be what is wanted. But your suggestion makes sense to me.